packages/enr/src/multiaddrs_codec.ts

```diff
@@ -20,7 +20,16 @@ export function decodeMultiaddrs(bytes: Uint8Array): Multiaddr[] {
     const multiaddrBytes = bytes.slice(index, index + size);
     index += size;
 
-    multiaddrs.push(multiaddr(multiaddrBytes));
+    let ma = multiaddr(multiaddrBytes);
+
+    // if the multiaddr contains `p2p`, it means that it supports `p2p-circuit`
```

Comment:
Please provide the full explanation as referred in the description of the PR (that `p2p-circuit` is not encoded due to space constraint in the ENR etc).